src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -226,6 +226,9 @@
   <data name="net_http_unix_https_support_unavailable_libcurl" xml:space="preserve">
     <value>The libcurl library in use ({0}) does not support HTTPS.</value>
   </data>
+  <data name="net_http_libcurl_newer_openssl_notsupported" xml:space="preserve">
+    <value>The libcurl library in use is built with {0}, we require version OpenSSL/1.0</value>
+  </data>
```

Comment:
https://github.com/Priya91/corefx-1/blob/abad3ab0b6b7f7bb8bb847b1cf29667e7c76dfe3/src/System.Net.Http/src/System/Net/Http/OSX/CurlHandler.SslProvider.cs#L26-L79 for how they got used.